src/app/dataset-block/metadata-block/components/yaml-view-section/yaml-view-section.component.html

```diff
@@ -0,0 +1,7 @@
+<div
+    class="container-content mt-4"
+    *ngIf="block && (isSetPollingSourceEvent || isSetTransformEvent)"
```

Comment:
There should be a single more general property, such as `isEventWithYamlView`, which internally would be equivalent to current condition.

Reply:
Fixed